dist/pom.xml

```diff
@@ -58,6 +58,10 @@
             312db,
             321db
         </databricks.buildvers>
+        <premergeUT.buildvers>
+            320,
```

Comment:
this doesn't have 3.1.1 in it so assume that it ested as base or something, we should add comment here

Reply:
Yeah, I would prefer to have some comments to explain list of shims and that we should cover the default shim test in default mvn verify command of premerge script